### PR TITLE
Task/bws92 master should return with array of the permission from repository should be unique only

### PR DESCRIPTION
**Pull request by Oleg-feriss**

### CODE REVIEW

File: backend/src/data/repositories/master/master.repository.ts [hunk 1]
```diff
@@ -23,7 +33,7 @@ class Master {
       return null;
     }
 
-    return Master.modelToEntity(master);
+    return Master.modelToEntity(master, ['']);
```
Comment (what1s1ove) [inline backend/src/data/repositories/master/master.repository.ts]:
```suggestion
    return Master.modelToEntity(master, []);
```

File: backend/src/data/repositories/master/master.repository.ts [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
+  Permission as PermissionM,
+} from '~/data/models/models';
 import { Master as MasterEntity } from '~/services/master/master.entity';
 
 type Constructor = {
   MasterModel: typeof MasterM;
+  PermissionModel: typeof PermissionM;
+};
+
+type Permission = {
```
Comment (what1s1ove) [inline backend/src/data/repositories/master/master.repository.ts]: move type from this file

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: backend/src/data/repositories/master/master.repository.ts [hunk 3]
```diff
@@ -23,7 +30,7 @@ class Master {
       return null;
     }
 
-    return Master.modelToEntity(master);
+    return Master.modelToEntity(master, []);
```
Comment (Yaroslaww-1) [inline backend/src/data/repositories/master/master.repository.ts]: Here you also need to fetch permissions (similarly to the getById)

**Review by Yaroslaww-1 — CHANGES_REQUESTED** [review]

File: backend/src/data/repositories/master/master.repository.ts (hunk 3 above)
Comment (what1s1ove) [reply backend/src/data/repositories/master/master.repository.ts]: +

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Review by Yaroslaww-1 — APPROVED** [review]

**Pull request merged**